apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -142,6 +147,7 @@ const CommitsAndReviewTab = props => {
       const newReview = await dataApi.openNewCodeReview(currentVersion);
       setOpenReviewData(newReview);
       setIsReadOnlyWorkspace(true);
+      setHasOpenCodeReview(true);
```

Comment:
I'm wondering if we need to set this in the refresh method as well. I think this would work as-is because it's only needed when the owner is viewing their own project. However, there is a case where the value wouldn't be correct. If a student is viewing their peer's project and their peer has an open code review `hasOpenCodeReview=true`, then the peer closes their review, the student hits refresh and the value hasn't changed `hasOpenCodeReview=true`. Functionally, it's fine right now, but it could be confusing for the developer why this value would only change for the owner of the code review.

I think there's a couple things we could do, either we make sure this value always represents whether the code review is open or closed for consistency, or we make a more specific variable, something like `displayOpenCodeReviewBanner` which is what this is actually used for and then we don't need to worry about updating the value if you're viewing a peers workspace. 

That was a lot. Let me know if that makes sense or if you want to jump on a call and talk about it